### DISCUSSION TITLE
size of undefined in bytes

### DISCUSSION THREAD

**Discussion by nearest-river** [denoland/deno#19809] [Q&A] [answered]
how much space does undefined take in the memory?..

**Comment by boywithkeyboard** [comment, marked as answer]
@kakashi-69-xd when you don't define a value for a variable, it's value is `undefined`.  So my understanding would be 0 bytes, however this StackOverflow thread considers something different: https://stackoverflow.com/questions/50854916/javascript-memory-impact-of-null-vs-undefined. This question is rather related to Google's V8 than Deno though.

Hope this helps! :)

**Reply by boywithkeyboard** [reply]
@kakashi-69-xd you can mark my response as the answer, so people know this thread is resolved!

**Reply by nearest-river** [reply]
do you know how the native Map works..
how it generates hashes?..

**Reply by boywithkeyboard** [reply]
no, i'm sorry 😕

**Reply by nearest-river** [reply]
> @kakashi-69-xd when you don't define a value for a variable, it's value is `undefined`.  So my understanding would be 0 bytes, however this StackOverflow thread considers something different: https://stackoverflow.com/questions/50854916/javascript-memory-impact-of-null-vs-undefined. This question is rather related to Google's V8 than Deno though.
> 
> Hope this helps! :)

cargo build --release
error: failed to get `deno_bindgen` as a dependency of package `xd v0.1.0 (/home/kakashi/Public/proton_xd/proton-xd)`

Caused by:
  failed to create directory `/home/kakashi/.cargo/registry/index/github.com-1ecc6299db9ec823`

Caused by:
  Permission denied (os error 13)

on vscode in linux
